lib/PodioLogger.php

```diff
@@ -28,7 +28,9 @@ public function enable()
 
     public function log($text)
     {
-        @mkdir(dirname($this->file));
+        if (!is_dir(dirname($this->file))) {
+            @mkdir(dirname($this->file));
```

Comment:
```suggestion
            mkdir(dirname($this->file, 0777, true));
```

a) the `@` should not be needed, as there are no warnings to suppress any more?
b) the recursive flag (true) should handle cases of nested folders as expected?